setup.cfg

```diff
@@ -1,3 +1,3 @@
 [pycodestyle]
-ignore = E402
-max-line-length = 80
+ignore = E121, E123, E126, E226, E24, E704, W503, E402
+max-line-length = 120
```

Comment:
Keeping the lines under 80 actually makes it more readable, because it forces you to better structure your code and longer lines are harder to read and comprehend. And it is more versatile, because it is better for people who code in terminal windows or use multiple panes in text editors and it helps when you look at git diffs on github for example. Big open source projects also advocate this, like Keras or Chromium.